pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
 
   <artifactId>acceptance-test-harness</artifactId>
-  <version>1.119-SNAPSHOT</version>
+  <version>${changelist}</version>
```

Comment:
I got a rid of the `1.`, but we may want to keep it to highlight future major changes properly 🤔

Reply:
not needed in this repo